packages/ns-api/files/ns.nathelpers.json

```
@@ -0,0 +1,13 @@
+{
+  "nat-manager": {
```

There is already a role named nat-manager inside `ns.nat.json`
```suggestion
  "nathelpers-manager": {
```